src/features/import-map.js

```diff
@@ -45,7 +46,15 @@ systemJSPrototype.resolve = function (id, parentUrl) {
 };
 
 function throwUnresolved (id, parentUrl) {
-  throw Error("Unable to resolve specifier '" + id + (parentUrl ? "' from " + parentUrl : "'"));
+  throw Error(errMsg(2, DEV ? "Unable to resolve specifier '" + id + (parentUrl ? "' from " + parentUrl : "'") : [id, parentUrl].join(', ')));
```

Comment:
This is unrelated to this PR, but since the error message is documenting it I think it is important to be specific - this is an `Unable to resolve bare specifier` error always.